modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_request.rb

```diff
@@ -89,10 +98,90 @@ def mark_replaced!(superseding_power_of_attorney_request)
 
     scope :unresolved, -> { where.missing(:resolution) }
     scope :resolved, -> { joins(:resolution) }
+
     scope :not_expired, lambda {
-      where.not(resolution: { resolving_type: 'AccreditedRepresentativePortal::PowerOfAttorneyRequestExpiration' })
+      where.not(
+        resolution: {
+          resolving_type: PowerOfAttorneyRequestExpiration.to_s
+        }
+      )
     }
 
+    ##
+    # This `concerning` block puts up some flashing lights around this
+    # complexity. It potentially wants to coexist directly with other model
+    # functionality or at a higher business logic layer, but extra care might be
+    # needed to pull that off without making a mess. This block is especially
+    # narrow--it only defines two scopes and no instance methods for example.
+    #
+    concerning :ProcessedScopes do
+      ##
+      # The 3x `LEFT OUTER JOIN`s with very particular join conditions make
+      # expressing both of the `processed` and `not_processed` relations easy
+      # to express with very simple `WHERE` conditions.
+      #
+      processed_join_sql_template = <<~SQL.squish
+        LEFT OUTER JOIN "ar_power_of_attorney_request_resolutions" "resolution" ON
+          "resolution"."power_of_attorney_request_id" = "ar_power_of_attorney_requests"."id"
+        LEFT OUTER JOIN "ar_power_of_attorney_request_decisions" "acceptance" ON
+          "resolution"."resolving_type" = :resolving_type AND
+          "resolution"."resolving_id" = "acceptance"."id" AND
+          "acceptance"."type" = :decision_type
+        LEFT OUTER JOIN "ar_power_of_attorney_form_submissions" "succeeded_form_submission" ON
+          "succeeded_form_submission"."power_of_attorney_request_id" = "ar_power_of_attorney_requests"."id" AND
+          "succeeded_form_submission"."status" = :submission_status
+      SQL
+
+      processed_join_sql =
+        ApplicationRecord.sanitize_sql(
+          [
+            processed_join_sql_template,
+            { resolving_type: PowerOfAttorneyRequestDecision,
+              decision_type: PowerOfAttorneyRequestDecision::Types::ACCEPTANCE,
+              submission_status: PowerOfAttorneyFormSubmission::Statuses::SUCCEEDED }
+          ]
+        )
+
+      ##
+      # `processed` and `not_processed` are the logical negation of one another,
+      # but this isn't enforced structurally in the code. An application of De
+      # Morgan's law is evident here. `invert_where` is a possibility to pull
+      # this off too, but it's not so usable because it inverts conditions that
+      # were chained prior.
+      #
```

Comment:
Can you remove the comment blocks in the file? That is not something we generally commit into the repo and the parts like `An application of De Morgan's law is evident here.` are unclear.